src/frontend/remix/index.ts

```diff
@@ -211,7 +211,7 @@ export class Remix extends KojiBridge {
   /**
    * Retrieves sensitive data that was [[encryptValue | stored as an encrypted value]].
    *
-   * NOTE: Only the creator of the Koji can access the decrypted value with this method. For example, to check that the value was entered correctly. To retrieve the value for other users, use {@doclink core-backend-secret#resolveValue | Secret.resolveValue} on the backend.
+   * NOTE: Only the creator who posted the Koji app in their profile can access the decrypted value with this method. For example, to check that the value was entered correctly. To retrieve the value for other users, use {@doclink core-backend-secret#resolveValue | Secret.resolveValue} on the backend.
```

Comment:
Remove reference to profile here. It's just the creator of the app (whether it's currently on their profile or not).

Reply:
"creator of the app" makes it easily confused with the developer of the app. What I understood is that we talk about three personas: the developer of the app, the social media creator who adds the app to their profile, and the user who interacts with the app. So "creator of the app" is vague.

Reply:
I thought we chose "creator" because they are social media creators, not because they are Koji app creators, which is false. The developers created the app; the "creators" only customized the app. Maybe we need a different word besides "creator".

Reply:
Maybe we can eliminate confusion in these package comments by only referring to a user's role in relationship to the app, rather than their role within the broader Koji ecosystem? 

At runtime, the app itself doesn't care whether the user is a developer-creator/social-media-creator/etc. (as each instance of the app is fully isolated from other instances) - would avoiding `creator` entirely standardizing on something like `owner` or `admin` make things more clear within the package docs?

Reply:
I like this perspective of discussing these values in relationship to the app. I do think we need to define what "owner" or "admin" mean in relation to how we discuss terms elsewhere in the docs. I believe "creator" is the correct term here because that is who can decrypt values on the frontend (whether they published this as a non-developer or a developer). Then, we can use "owner" as the technical term for the reference docs.

Maybe something like "Only the Koji creator ("owner") can ..."

Reply:
I like "owner", which can be either the developer or the social-media creator.

We already use "admin" as a special case of "user", i.e., someone who can access the admin context if there is one in the app.

I'll add "owner" in the glossary as someone who owns a specific version of the app. The developer owns the original version of the app; the creator who customized the app owns that specific customized version.

Reply:
Even though it might not be the case long term (e.g., if there end up being "teams" or "organizations"), today, the admin and owner user will always be the same. I think that's going to be the case for the foreseeable future, so I wouldn't be opposed to using a single term for internal documentation.

From the app's perspective, it only ever has one "creator" - the user who owns that App ID in Koji's Database. That "creator" might be the person who wrote the original code, or it might be the person who customized it; but by design there is no case where an app needs to understand that it might have other clones, or a parent, that are owned by different users.

Reply:
I think it would create more confusion if we use the same term ("creator") to mean two different things within the same documentation set. These TypeDoc docs end up in the same website as the AsciiiDoc docs where we use "creator" to mean the people who customize the app and "developer" to mean the coder of the original app.

Maybe we need a different word to mean "customizer". One that's more elegant than "customizer".